lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -1,11 +1,19 @@
+require 'ddtrace/contrib/rails/utils'
+require 'ddtrace/ext/ruby'
+require 'ddtrace/ext/sql'
+require 'ddtrace/ext/app_types'
+
 module Datadog
   module Contrib
     module ActiveRecord
       # Patcher enables patching of 'active_record' module.
       module Patcher
         include Base
         register_as :active_record, auto_patch: false
-        option :service_name
+        option :service_name do |value|
+          value.tap { @database_service_name = nil }
+        end
+        option :ruby_service_name
```

Comment:
I'm not entirely sure to keep `ruby_` as a prefix, since it's too generic. If we check what this integration is, we can state that `ActiveRecord` is an architectural pattern that stores in-memory object data in relational databases ([ref](https://en.wikipedia.org/wiki/Active_record_pattern)). In the case of Rails, this pattern is used to implement the ORM. I think we can assume that the most common case is to use it to store data in-memory (or somewhere), so we may use different names such as:
* `active_record_name`
* `orm_service_name` (I prefer this one because it keeps the `service_name` suffix that is clear in our terminology + `orm_` has a specific meaning that is generic enough to cover instantiations and any other operation we may have in the future)

What do you think @delner ?

Reply:
I like `orm_service_name`. Ill change that.

Reply:
one thing I forgot in the review is that we need to update also the documentation with the new option